src/site/markdown/server/manage-webadmin.md

```diff
@@ -358,7 +358,45 @@ Response codes:
 
 The kind of task scheduled depends on the action parameter. See below for details.
 
-### Recomputing JMAP fast message view projection
+#### Fixing mailboxes inconsistencies
+
+This task is only available on top of Guice Cassandra products.
+
+```
+curl -XPOST /mailboxes?task=solveInconsistencies
+```
+
+Will schedule a task for fixing the mailbox projection.
+
+[More details about endpoints returning a task](#Endpoints_returning_a_task).
+
+The scheduled task will have the following type `solve-mailbox-inconsistencies` and the following `additionalInformation`:
+
+```
+{
+  "type":"solve-mailbox-inconsistencies",
+  "processedMailboxEntries": 3,
```

Comment:
you don't report on fixed things?

Reply:
That what `fixedInconsistencies` is about. Processed intent  is showing progress.

Or do you want a list of fixed ids?

Reply:
The use case is :  
1. bob asks his admin why there's a mailbox missing (or whatever inconsistencies is about)
2. admin run the solve-mailbox-inconsistencies task in the hope of fixing it
3. admin don't have the information in the report to know if a mailbox for this user has been fixed

Reply:
So we want the list of fixed ids ;-)